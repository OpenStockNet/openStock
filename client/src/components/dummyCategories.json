[
  {
    "_id": 1,
    "name": "Browser",
    "icon": "https://via.placeholder.com/40"
  },
  {
    "_id": 2,
    "name": "Search Engine",
    "icon": "https://via.placeholder.com/40"
  },
  {
    "_id": 3,
    "name": "Messenger",
    "icon": "https://via.placeholder.com/40"
  },
  {
    "_id": 4,
    "name": "E-Mail Provider",
    "icon": "https://via.placeholder.com/40"
  },
  {
    "_id": 5,
    "name": "Online Streaming",
    "icon": "https://via.placeholder.com/40"
  },
  {
    "_id": 6,
    "name": "Navigation",
    "icon": "https://via.placeholder.com/40"
  }
]
