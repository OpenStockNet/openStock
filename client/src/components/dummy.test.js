import dummyCategories from './dummyCategories.json';

test ('cats are correct', () => {
    expect(dummyCategories).toMatchSnapshot();
})