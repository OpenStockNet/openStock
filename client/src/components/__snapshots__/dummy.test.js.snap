// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`cats are correct 1`] = `
Array [
  Object {
    "_id": 1,
    "icon": "/images/categories/iconBrowser.svg",
    "name": "Browser",
  },
  Object {
    "_id": 2,
    "icon": "/images/categories/iconSearchEngine.svg",
    "name": "Search Engine",
  },
  Object {
    "_id": 3,
    "icon": "/images/categories/iconMessenger.svg",
    "name": "Messenger",
  },
  Object {
    "_id": 4,
    "icon": "/images/categories/iconEMail.svg",
    "name": "E-Mail Provider",
  },
  Object {
    "_id": 5,
    "icon": "/images/categories/iconOnlineStreaming.svg",
    "name": "Online Streaming",
  },
  Object {
    "_id": 6,
    "icon": "/images/categories/iconNavigation.svg",
    "name": "Navigation",
  },
]
`;
